### ISSUE TITLE
Migration script breaks non-Latin translations

### ISSUE THREAD

**Issue by bear-in-a-box** [onfido/onfido-sdk-ui#1238]
### What was the expected behavior?

Translations for languages with characters outside of the basic Latin range (Polish, Chinese), pushed through the migration script are valid are working.

### What happened instead?

> Uncaught (in promise) DOMException: Failed to execute 'btoa' on 'Window': The string to be encoded contains characters outside of the Latin1 range.

### Version info:

- Onfido SDK 6.2.0
- OS macOS Catalina 10.15.7
- Browser Google Chrome 86.0.4240.111
- Device N/A
- Node 12.13.1
- Framework or relevant dependencies (if applicable) N/A

### Integration configuration:

N/A

### Steps to reproduce:

1. Prepare raw JSON file with translations with schema required by the library
2. Run the migration script on these JSON files
3. Import these files into a project
4. Init Onfido SDK UI with the object parsed from one of the JSON files

### Demo:

N/A

### Additional info:

- Raw JSON files that I've got prepared were working (before version bump) properly.
- Files, when opened in a text editor (VS Code for instance) don't seem to be broken.

**Comment by ethan605** [comment]
Hi @bear-in-a-box, thanks for your information. Can I please ask for the raw JSON file that you prepared to run with the migration script?

**Comment by bear-in-a-box** [comment]
Sorry, NDA doesn't permit me to do so publicly.

I've got a new observation, though. Translations with UTF-8 encoding and non-Latin characters fail instantly after only bumping the Onfido SDK UI version from 5.13 to 6.2. They're the same JSON files.

**Comment by ethan605** [comment]
Got it @bear-in-a-box.

Though I can't gather enough contexts to reproduce at my end, there're several suggestions:
* Can you please try upgrading from `v5.13.0` to `v6.0.0` first, then `v6.1.0` to run `migrate_locales` script, then `v6.2.0`?
* If it doesn't help, can you please provide a sample JSON file with some translation copies containing non-Latin characters (which is similar to your file)?

**Comment by bear-in-a-box** [comment]
I've managed to prepare an example for you. :)

```
{
  "locale": "en",
  "phrases": {
    "welcome": {
      "description_p_1": "ąęśćłź",
      "description_p_2": "ąęśćłź",
      "next_button": "ąęśćłź",
      "title": "ąęśćłź"
    }
  }
}
```

Please simply attach that to the init method. It should crash the library.

**Comment by ethan605** [comment]
Hi @bear-in-a-box, I'd like to update with you that this issue is being fixed and we will release a new version this week.
Thank you

**Comment by stefaniacardenas** [comment]
Hi, this has been fixed in version `6.2.1`. Feel free to reopen or create another issue if you need any further help.
